### ISSUE TITLE
Provide a way to easily disable Disqus.

### ISSUE THREAD

**Issue by xcellardoor** [dlecina/StayPuft#17]
I've managed to do it manually by just taking out all of the footer sections and page scripts which use Disqus, but providing an easy toggle in a config or something would be great.

**Comment by dlecina** [comment]
I agree, this would be an improvement. I'd also like a different toggle for Disqus in _pages _(as opposed to _posts_). I'll have to think about the best way to do this. Any suggestions would be appreciated.

Honestly I think Ghost should provide some sort of API for users to configure themes inside the admin section, but we'll have to make do with what we have.

**Comment by dlecina** [comment]
I'm still waiting for Ghost to provide a better way to do this (the [@config helper](https://themes.ghost.org/docs/config) seems to be the prime candidate), but you can now disable disqus fairly easily. There are only two lines you have to remove. The relevant information has been added to the [FAQ](https://github.com/dlecina/StayPuft#faq):

3. **How can I disable comments?**

  To disable comments, simply remove the line `{{> "post-comments"}}` from `post.hbs` and `page.hbs`. You do not need `disqus.hbs` in this case.

4. **How can I disable comments only on pages?**

  To disable comments only on pages, simply remove the line `{{> "post-comments"}}` from `page.hbs`.

I will leave the issue open, however, until a single-line solution is available.
